commit 93fa72519a21dcc18758e31a5408b6f5da325795 (refs/changes/65/1465/6)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-02-24T01:54:30+00:00 (2 years, 7 months ago)
    
    PUBAPI-1361 update cueball dep to latest
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index 720149a..0a4405a 100644
--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
         "nopt": "2.0.0",
         "restify": "4.3.0",
         "bunyan": "1.8.1",
-        "cueball": "1.1.9",
+        "cueball": "2.1.1",
         "kang": "1.1.0",
         "sdc-clients": "10.0.4",
         "ufds": "1.2.0",
